### DISCUSSION TITLE
Any mapping of metrics with compatible classifiers?

### DISCUSSION THREAD

**Discussion by prabhant** [online-ml/river#547] [Q&A] [answered]
Hi, during a recent experiment I noticed that all metrics are not compatible with all classifiers. is there any mapping for this? It will save quite some time.

**Comment by MaxHalford** [comment, marked as answer]
Do you have an example? There's no mapping per say but each metric has a `works_with` method.

```py
from river.linear_model import LogisticRegression
from river.metrics import Accuracy

model = LogisticRegression()
metric = Accuracy()
print(metric.works_with(model))
```

**Reply by prabhant** [reply]
That was helpful, thanks :)
